### PR TITLE
Add game to event 

### PR DESCRIPTION
**Pull request by ShouldIStayOrShouldIGo**
Add game from game list to Event form. Frontend fix

### CODE REVIEW

**Comment by hirokiraj** [comment]
Quite large one, 1,5k changed lines count is a little bit too much for a one PR

**Comment by ShouldIStayOrShouldIGo** [comment]
@hirokiraj  some changes were made by Rafał so in reality the number of changed lines is under 1k
 \ (•◡•) /

File: spec/controllers/events_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe EventsController, type: :controller do
+
```
Comment (hirokiraj) [inline spec/controllers/events_controller_spec.rb]: fill controller tests here

File: spec/factories/events.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :event do
+    title { "MyString" }
```
Comment (hirokiraj) [inline spec/factories/events.rb]: use faker to randomize data in factories

File: spec/helpers/events_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# describe EventsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe EventsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (hirokiraj) [inline spec/helpers/events_helper_spec.rb]: remove unused code

File: spec/models/event_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (hirokiraj) [inline spec/models/event_spec.rb]: provide tests for a model

File: app/views/events/show.json.jbuilder [hunk 5]
```diff
@@ -0,0 +1,2 @@
+json.title @event.title
```
[the remark this answers is not in the document]
Comment (hirokiraj) [reply app/views/events/show.json.jbuilder]: `json.call @event, :title, :description`

File: app/views/games/_form.haml [hunk 6]
```diff
@@ -1,18 +1,38 @@
...
-
-    = form.submit 'Add to my collection'
-  
\ No newline at end of file
+= form_for @game do |form|   
+    .cotainer
+        .row.justify-content-center
+            .col-md-8
+                .card.bg-transparent#menu_card
+                    .card-body.container{style: "background-color:  rgba(4, 230, 255, 0.5); border-radius:10px;"}
```
Comment (Vakrim) [inline app/views/games/_form.haml]: prefer adding new classes and styling them in CSS that writing inline styles

File: app/assets/stylesheets/events.scss [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+  background-color: rgba(255, 255, 255, 0.75);
+}
+.events-header {
+  width: 60%;
+  margin: 15px;
+  width: 100%;
+  text-align: center;
+  color: #510280;
+}
+#col_en{
```
Comment (Vakrim) [inline app/assets/stylesheets/events.scss]: prefer using classes for styling instead of ids

File: app/javascript/packs/events.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import { React, ReactDOM, Component } from "react";
```
Comment (Vakrim) [inline app/javascript/packs/events.js]: remove this file if it is not used

File: app/assets/stylesheets/events.scss [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+  width: 50%;
+  margin-left: -11%;
+}
+#col_event_name{
+  width: 50%;
+  margin-left: 0%;
+}
+#event_name label{
+  width: 119%;
+  margin-top: 1.1%;
```
Comment (Vakrim) [inline app/assets/stylesheets/events.scss]: I see a lot of different percentages values used for sizes and positioning. I'm afraid that so many values may be difficult to maintain and implement responsiveness for mobile devices in future if needed. Try to utilize bootstrap layouts for this job and reduce custom styles to minimum in future.

**Pull request merged**